spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :user do
+    email { "MyString" }
+    password { "" }
```

Comment:
is this a valid user?

Reply:
> is this a valid user?

i think in case if i want to create users I wont have to check their individual passwords if i were to log them in